### ISSUE TITLE
Implement new API function 'Get a member of a group or project, including inherited members'

### ISSUE THREAD

**Issue by RH16** [gitlab4j/gitlab4j-api#557]
According to GitLab API docu this was added in GitLab 12.4 and I would need exactly that.

**Comment by gmessner** [comment]
@RH16 Will add this to the next release which is scheduled for Saturday 5/2/2020

**Comment by gmessner** [comment]
@RH16 ```getMember()``` methods have been added to provide this functionality:

In ProjectApi:
```
public Member getMember(Object projectIdOrPath, Integer userId, Boolean includeInherited)
public Optional<Member> getOptionalMember(Object projectIdOrPath, Integer userId, Boolean includeInherited)
```
In GroupApi:
```
public Member getMember(Object groupIdOrPath, Integer userId, Boolean includeInherited)
public Optional<Member> getOptionalMember(Object groupIdOrPath, Integer userId, Boolean includeInherited)
```
The ```userId``` parameter is optional and can be ```null```.

This is available in release 4.14.28, please verify and close this issue when you have time.

**Comment by RH16** [comment]
Works fine. Thanks alot, Greg!!
